sentinelsat/scripts/cli.py

```diff
@@ -250,9 +250,13 @@ def cli(
         end = end or "NOW"
         products = api.query(date=(start, end), order_by=order_by, limit=limit, **search_kwargs)
 
-    if footprints is True:
+    if footprints is not None:
+        if os.path.isdir(footprints) or footprints[len(footprints) - 1] == os.path.sep:
```

Comment:
`footprints[len(footprints) - 1] == os.path.sep` this would fail if a user does not provide a trailing slash.

works: `/some/path/`
fails: `/some/other/path`

Reply:
That's what `os.path.isdir(footprints)` is for. The other part is for checking whether the user is specifying a directory that does not exist. If the trailing slash is left out, it will use that part as the file to write to. Does that clear things up?

Reply:
I don't think we need special treatment for trailing slashes in paths? `os.path.join` treats both the same and so does `os.path.isdir`. So you can just remove that `or` condition.

Reply:
I added it for non-existent directories to be added, but I'll remove it, then.